util/shared/json-schema.mjs

```diff
@@ -76,9 +76,17 @@ const addExternalMarkdown = obj => {
 // DOES NOT DEAL WITH ERRORS
 const getSchemaContent = fileStream => fileStream
     .filter(filepath => path.extname(filepath) === '.json')
+    .flatMap(loadAndParseSchema)
+
+const loadAndParseSchema = filepath => h.of(filepath)
     .flatMap(fsReadFile)
     .map(bufferToString)
     .map(JSON.parse)
```

Comment:
Okay, so if `JSON.parse` throws an error, you deal with it in the handler and terminate the process?

Reply:
You can optionally just do the `console.error` and `push(null, {})` and let the program keep running. Or do you want a CI process to fail on a single error?

Reply:
Was going for a single error and exit, but now that they are pretty i think you are right.

Reply:
Okay, I bubbled up the errors, and now list all files with errors in them.